andela/ah-frontend-poseidon
# #162162973 Users should be able to share articles on social media and via email.

**What does this PR do?**

Add share buttons to each article that share that article.  

**Description of the tasks to be completed?**
- Add share button for facebook. 
- Add share button for twitter. 
- Add share button for email. 

**How should this be manually tested?**
- Clone the repository
- git checkout ft-dashboard-163081040
- install dependencies using `yarn install`
- Run the app using `yarn start`
- The application should be running.
- Open browser and go to http://localhost:3000
- Select an article.
- Click on a share button of yourchoice. 

**What are the relevant pivotal tracker stories?**
[#162162973](https://www.pivotaltracker.com/story/show/162162973)

**Screenshots**
![image](https://user-images.githubusercontent.com/17021137/51755707-6453a300-20d0-11e9-8620-c30dc43236a6.png)
